ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__cnf-network-4.17.yaml

```diff
@@ -18,6 +18,7 @@ tests:
   steps:
     env:
       CLUSTER_NAME: kni-qe-92
+      OPERATORS: '[{"name":"sriov-network-operator","catalog":"redhat-operators-stage","nsname":"openshift-sriov-network-operator","deploy_default_config":"true"},{"name":"ptp-operator","catalog":"redhat-operators-stage","nsname":"openshift-ptp","ns_labels":{"workload.openshift.io/allowed":"management","name":"openshift-ptp"}},{"name":"kubernetes-nmstate-operator","catalog":"redhat-operators","nsname":"openshift-nmstate","deploy_default_config":"true"},{"name":"sriov-fec","catalog":"certified-operators","nsname":"vran-acceleration-operators","channel":"stable"},{"name":"metallb-operator","catalog":"redhat-operators-stage","nsname":"metallb-system","channel":"stable","og_spec":{},"deploy_default_config":"true"}]'
```

Comment:
This is ok. Another way might be [this one](https://github.com/ccardenosa/release/blob/master/ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__ci-4.19.yaml#L48-L54). But maybe you already know it and you just perfer the current format.